src/modules/feature-modules/innovator/services/innovator.service.ts

```diff
@@ -37,6 +37,20 @@ export type GetOwnedInnovations = {
   expirationTransferDate: DateISOType | null;
 };
 
+export enum InnovationArchiveReasonEnum {
+  // User achive reasons
+  DEVELOP_FURTHER = 'DEVELOP_FURTHER',
+  HAVE_ALL_SUPPORT = 'HAVE_ALL_SUPPORT',
+  DECIDED_NOT_TO_PURSUE = 'DECIDED_NOT_TO_PROCEED',
+  ALREADY_LIVE_NHS = 'ALREADY_LIVE_NHS',
+  OTHER_DONT_WANT_TO_SAY = 'OTHER_DONT_WANT_TO_SAY',
+  // Automatic archive reasons
```

Comment:
I would include only the User archive reasons as the others aren't sent nor read by the frontend